docs/source/guides/auth.mdx

```diff
@@ -243,6 +243,23 @@ Before continuing, you need to set up the Auth0 client to accept an additional c
 
 </ExpansionPanel>
 
+### Optional: Connecting to Apollo MCP Server from Claude
+
+In order to get the full experience of using and MCP server authorization, you'll want to use an LLM of some sort. Anthropic's Claude provides an accessible method of connecting to an MCP server, through [their Custom Connector feature](https://support.anthropic.com/en/articles/11175166-getting-started-with-custom-connectors-using-remote-mcp).
+
+If you'd like to connect your MCP server with Claude, the first thing you need to do is deploy your server behind HTTPS. This is because Claude requires all custom connectors to have URLs that begin with `https`. Whether you do this by providing a local SSL certificate, or deploying to a staging or production environment, is up to you.
```

Comment:
Let's try to be a bit more helpful here. For example we can link to our [deployment docs](https://www.apollographql.com/docs/apollo-mcp-server/guides#deploying-the-mcp-server).

And find another one for "providing a local SSL certificate".